components/mpas-ocean/src/shared/mpas_ocn_vmix.F

```diff
@@ -288,13 +287,10 @@ subroutine ocn_vel_vmix_tend_implicit_rayleigh(dt, kineticEnergyCell, vertViscTo
         N = maxLevelEdgeTop(iEdge)
         if (N .gt. 0) then
 
-         ! layerThicknessEdge is computed in compute_solve_diag, and is not available yet,
-         ! so recompute layerThicknessEdge here.
```

Comment:
Noting that this statement is false. `layerThicknessEdge` is computed in `ocn_diagnostic_solve,` which is called for `timeLevel=2` in each time integration scheme before `ocn_vel_vmix_tend` is called.